api/src/test/java/org/openmrs/api/context/ContextTest.java

```diff
@@ -82,7 +82,40 @@ public void authenticate_shouldNotAuthenticateWithNullUsername() {
 	 */
 	@Test(expected = ContextAuthenticationException.class)
 	public void authenticate_shouldNotAuthenticateWithNullUsernameAndPassword() {
-		Context.authenticate(null, null);
+		Context.authenticate((String) null, null);
+	}
+	
+	/**
+	 * @see Context#authenticate(String,String)
+	 */
+	@Test(expected = ContextAuthenticationException.class)
+	public void authenticate_shouldNotAuthenticateWithNullCredentials() {
```

Comment:
Is this testing against only the credentials being null or both credentials and authentication scheme?